templates/open-enterprise/temp/scripts/test-ganache.sh

```diff
@@ -33,7 +33,7 @@ setup_testing_variables() {
 
 start_ganache() {
   echo "Starting ganache-cli..."
-  npx ganache-cli -i ${NETWORK_ID} -l ${GAS_LIMIT} -a ${ACCOUNTS} -e ${BALANCE} -p ${PORT} > /dev/null &
+  npx aragon devchain --reset -a ${ACCOUNTS} -p ${PORT} > /dev/null &
   pid=$!
   sleep 3
   echo "Running ganache-cli with pid ${pid} in port ${PORT}, gas limit set to: ${GAS_LIMIT}"
```

Comment:
Same here, especially since we arn't setting a gas-limit anymore.

Reply:
I am sorry, but I don't understand what is the purpose of this change.

I would like to know if there are clear benefits for this, I initially see some downsides, for example, forcing to reset the local blockchain every time a test is run.

We probably shouldn't rely into `aragon devchain` for tests, one possible reason is the pre-deployed snapshot that is uses, it defeats truffle clean room features for contracts. In the other hand we are forced to reset the local devchain, I think we should not enforce the developer to do this, because there exist different workflows that might be altered. In my case, sometimes I find it convenient to reuse template or other contracts deployments during development, so I would not like to be forced to reset the data when there are better solutions.

I asked @PeterMPhillips the other day to fix template test that was not working because a problem with gas during standard bounties deployment, I am not sure if this was  the kind of solution I expected, I would like to know what specific problem is this PR fixing.

I fixed the template test issue in 987b57b7e4468d46b1efa9ab0d69c8e370bb6f2d

Reply:
Maybe using `aragon devchain` eventually saves us from running the deployments scripts before tests, but in this case this PR would be incomplete until that solution is tested, and those deployment scripts removed. But I don't really think we should merge this as it is, because we are not using any `aragon devchain` pre-deployed asset in out tests, so this PR as it is does not make lot of sense, changing the command alone does not improve or solve anything, unless I am missing something, in that case, please correct me.